### PR TITLE
LPS-131604 translations auto translation by field JSP markup

### PR DESCRIPTION
**Pull request by boton**
Add placeholder auto-translation markup on the server render.

**Test Plan:**

- Enable `com.liferay.translation.web.internal.configuration.FFAutoTranslateConfiguration` feature flag.
 
   ```
   cd LIFERAY_PORTAL_PATH
   echo "enabled=B\"true\"" > ../bundles/osgi/configs/com.liferay.translation.web.internal.configuration.FFAutoTranslateConfiguration.config
  ```
  
- Configuration: System settings > Content and Data > category.translations (default cog icon)
- Enable and add well-formed JSON or a valid key
- Disable JS in your browser (in chrome: https://developer.chrome.com/docs/devtools/javascript/disable/) avoiding that placeholder markup being hydrated by the react component.

---

**Server rendering (with js disabled):**

image

**Client rendering:**

image

### CODE REVIEW

**Comment by boton** [comment]
ci:test:sf

**Comment by boton** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

File: modules/apps/translation/translation-web/src/main/resources/META-INF/resources/translate.jsp [hunk 1]
```diff
@@ -136,56 +140,90 @@ renderResponse.setTitle(translateDisplayContext.getTitle());
...
-												</div>
-											</c:when>
-											<c:otherwise>
-												<aui:input dir='<%= LanguageUtil.get(translateDisplayContext.getTargetLocale(), "lang.dir") %>' disabled="<%= true %>" label="<%= label %>" name='<%= "infoField--" + infoField.getName() + "--" %>' type='<%= multiline ? "textarea" : "text" %>' value="<%= targetContent %>" />
-											</c:otherwise>
-										</c:choose>
-									</clay:col>
-								</clay:row>
+								<liferay-util:buffer
+									var="ColContentBuffer"
```
Comment (carloslancha) [inline modules/apps/translation/translation-web/src/main/resources/META-INF/resources/translate.jsp:144]: AFAIK this should be lower camelCase: `colContentBuffer`. Anyway I think we could find a better name for this that represent what it's the content, not where is going to be rendered.

**Review by carloslancha — COMMENTED** [review]
LGTM overall, just a minor nit about naming.

File: modules/apps/translation/translation-web/src/main/resources/META-INF/resources/translate.jsp (hunk 1 above)
Comment (boton) [reply modules/apps/translation/translation-web/src/main/resources/META-INF/resources/translate.jsp:144]: Thanks, you are right. It's not even a column but a row. Maybe `contentRowBuffer` fits better?

Comment (carloslancha) [reply modules/apps/translation/translation-web/src/main/resources/META-INF/resources/translate.jsp:144]: Again we're describing where is going to be rendered (a row), not what. And I don't think `buffer` really needs yo be included in the name.

Comment (boton) [reply modules/apps/translation/translation-web/src/main/resources/META-INF/resources/translate.jsp:144]: Only `translateRow` comes to mind but is more of the same thing. If I try to put something like `field`, `preview` or ` editor` it seems like I mean smaller components. 
Do you have any specific suggestions?

Comment (boton) [reply modules/apps/translation/translation-web/src/main/resources/META-INF/resources/translate.jsp:144]: Go with `translationContent`!

**Comment by boton** [comment]
ci:forward

**Pull request closed without merging**